integration/benchmark/load/load_type_test.go

```diff
@@ -28,6 +27,7 @@ import (
 	"github.com/onflow/flow-go/integration/benchmark/common"
 	"github.com/onflow/flow-go/integration/benchmark/load"
 	"github.com/onflow/flow-go/integration/convert"
+	"github.com/onflow/flow-go/integration/emulator"
```

Comment:
Maybe it might be good to rename this, so it doesn't get confused with the "real" Flow Emulator.

I can't think of a good name though

Reply:
I'm going to add a package-level godoc adding some context about what this is and how it relates to the "real" Emulator. 

Regarding getting confused with the "real" Emulator, we could also make it an internal package, so it can't be imported from any module besides `flow-go/integration`. That at least limits the possible confusion surface area to developers working on the integration test package.

Reply:
Both would be great, thanks!